matchers/README.md

```diff
@@ -0,0 +1,160 @@
+# OnPar Matchers
+
+OnPar provides a set of minimalistic matchers to get you started.
+However, the intention is to be able to write your own custom matchers so that
+your code is more readable.
+
+
+## Matchers List
+- [String Matchers](#string-matchers)
+- [Logical Matchers](#logical-matchers)
+- [Error Matchers](#error-matchers)
+- [Channel Matchers](#channel-matchers)
+- [Collection Matchers](#collection-matchers)
+- [Other Matchers](#other-matchers)
+
+
+## String Matchers
+### StartWith
+StartWithMatcher accepts a string and succeeds if the actual string starts with
+the expected string.
+
+```go
+Expect(t, "foobar").To(StartWith("foo"))
+```
+
+### EndWith
+EndWithMatcher accepts a string and succeeds if the actual string ends with
+the expected string.
+
+```go
+Expect(t, "foobar").To(EndWith("bar"))
+```
+
+### ContainSubstring
+ContainSubstringMatcher accepts a string and succeeds if the expected string is a
+sub-string of the actual.
+
+```go
+Expect(t, "foobar").To(ContainSubstring("ooba"))
+```
+
+## Logical Matchers
+### Not
+NotMatcher accepts a matcher and will succeed if the specified matcher fails.
+
+```go
+Expect(t, false).To(Not(BeTrue()))
+```
+
+### BeAbove
+BeAboveMatcher accepts a numerical value. It succeeds if the actual is greater
```

Comment:
Really the value must be a float64 and not any numeric. This is something we might want to support though.